torch/csrc/distributed/rpc/init.cpp

```diff
@@ -232,6 +232,9 @@ PyObject* rpc_init(PyObject* /* unused */) {
   // pythonRpcHandler is cleaned up in shutdown(), after
   // shutdown(), python objects returned from rpc python call can not be
   // resolved.
+  // TODO Once python object can be tagged as IValue and c10::ivalue::Future is
+  // implemented as generic Future<IValue>, we can consider all rpc call
+  // to return a future<IValue> later on.
   auto future = shared_ptr_class_<FutureMessage>(module, "Future")
```

Comment:
I am thinking about Yanli's merging request.
This can only be merged after "_invoke_rpc_builtin" and "_invoke_rpc_python_udf" returns a `ivalue::Future` wrapped by `torch::jit::PythonFutureWrapper` directly rather than wrapping a `ivalue::Future` into a `utils::Future<Message>`?

Reply:
yeah will do that i a follow up PR

Reply:
I mean to merge line 807 in torch/csrc/jit/python/init.cpp (somehow I can not inline comment on that line)